rmd/Story 1.json

```diff
@@ -616,393 +616,393 @@
   },
   "101010020120051": {
     "OriginalText": "フィルディア隊長！ 緊急事態です！",
-    "Text": "",
-    "Enabled": false
+    "Text": "Captain Fildia! There's an emergency!",
+    "Enabled": true
   },
   "101010020130004": {
     "OriginalText": "どうした？",
-    "Text": "",
-    "Enabled": false
+    "Text": "What happened?",
+    "Enabled": true
   },
   "101010020140051": {
     "OriginalText": "先遣隊がダーカーと接触、\n交戦を開始したのですが、\nフォトンが！ フォトンが使えません！",
-    "Text": "",
-    "Enabled": false
+    "Text": "The survey team has encountered a group of Darkers!\nAfight broke out but...\nThey- they can't use Photon!
+    "Enabled": true
   },
   "101010020150004": {
     "OriginalText": "何を言っているんだ？",
-    "Text": "",
-    "Enabled": false
+    "Text": "What!?",
+    "Enabled": true
   },
   "101010020160051": {
     "OriginalText": "武器もフォトンアーツも使えないんです！\nマグナス艦長を中心に防衛線を展開し、\n撤退を開始していますが、長くは保ちません！",
-    "Text": "",
-    "Enabled": false
+    "Text": "Their weapons are no good too!/nThey're holding a defensive line around the Captain and attempting retreat,\n but they won't last long like this!",
```

Comment:
Make sure there are no spaces around any \n newlines, it'll offset the text and make it look weird